### ISSUE TITLE
config.cson no longer support object

### ISSUE THREAD

**Issue by RustJason** [gwomacks/php-debug#64]
```
"php-debug":
    CustomExceptions: [
      "t"
    ]
    PathMaps: [
      {
        local: "G:\\projects\\"
        remote: "/home/gabriel/projects/"
      }
    ]
```
Those code will be modified automatically
```
"php-debug":
    CustomExceptions: [
      "t"
    ]

**Comment by colbydehart** [comment]
This is also true for me. Ubuntu 14.04. Atom V1.0.0

**Comment by theseamusjames** [comment]
Throws an error on OSX too. `[stdin]:9:13: error: unexpected newline
"php-debug":`

for 

```
"php-debug":
{
  ServerPort: 9001
}

**Comment by donCarlosOne** [comment]
I've found that manually typing it in (not copy paste) at the end of config.cson allows the config.cson file to be saved.

**Comment by lucascaro** [comment]
Check out the fix in #85, that did the trick for me.
